modules/vaos/app/services/vaos/v2/appointments_service.rb

```diff
@@ -714,6 +717,35 @@ def set_cancellable_false(appointment)
         appointment[:cancellable] = false
       end
 
+      def set_modality(appointment)
+        modality = nil
+        if appointment[:service_type] == 'covid'
+          modality = 'vaInPersonVaccine'
+        elsif appointment.dig(:service_category, 0, :text) == 'COMPENSATION & PENSION'
+          modality = 'claimExamAppointment'
+        elsif appointment[:kind] == 'clinic'
+          modality = 'vaInPerson'
+        elsif appointment[:kind] == 'telehealth'
+          modality = telehealth_modality(appointment)
+        elsif appointment[:kind] == 'phone'
+          modality = 'vaPhone'
+        elsif appointment[:kind] == 'cc' && !appointment[:start].nil?
```

Comment:
Just curious why is `start` being checked specifically for type `cc` appointments only?